folio-spring-i18n/src/main/java/org/folio/spring/model/TranslationMatchQuality.java

```diff
@@ -0,0 +1,59 @@
+package org.folio.spring.model;
+
+import java.util.Locale;
+import org.folio.spring.model.TranslationFile.LanguageRegionPair;
+
+/**
+ * How good a {@link TranslationMap TranslationMap} relates to a given {@link java.util.Locale Locale}.
+ */
+public enum TranslationMatchQuality {
+  /** If the paired translation is a perfect match for language and country. */
+  PERFECT_MATCH,
+  /** If the paired translation is a match for language only and not country (or not applicable to country). */
+  LANG_ONLY,
+  /** If the paired translation is not a match for language or country (should be the server's default locale). */
```

Comment:
Maybe rather than say that this should be the server's default locale, describe the fallback if there's another layer to fallback to before the server default?